app/components/primer/alpha/overlay/body.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    class Overlay
+      # A `Overlay::Body` is a compositional component, used to render the
+      # Body of an overlay. See <%= link_to_component(Primer::Alpha::Overlay) %>.
+      class Body < Primer::Component
+        # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+        def initialize(**system_arguments)
+          @system_arguments = deny_tag_argument(**system_arguments)
+          @system_arguments[:tag] = :div
+          @system_arguments[:classes] = class_names(
```

Comment:
Body is missing a `padding` param as detailed here https://github.com/github/primer/blob/main/apis/overlay-api.md#overlaybasebody

```
.Overlay-body.Overlay-body--paddingCondensed
.Overlay-body.Overlay-body--paddingNone
```